OloEngine/src/OloEngine/Scripting/VisualScript/VisualScriptGraph.cpp

```
@@ -0,0 +1,95 @@
+#include "OloEnginePCH.h"
+#include "VisualScriptGraph.h"
+
+#include <algorithm>
+
+namespace OloEngine::VisualScript
+{
+    const std::string& VisualScriptNode::GetProperty(std::string_view key, const std::string& fallback) const
```

Cppcheck (2.21.0)

[style] The function 'GetProperty' is never used.

(unusedFunction)